### DISCUSSION TITLE
请教模糊音怎么设置？

### DISCUSSION THREAD

**Discussion by fjlionj** [tumashu/pyim#444] [Q&A] [answered]
设置模糊音。可以通过设置 `pyim-pinyin-fuzzy-alist’ 变量来自定义模糊音。

能否给一个例子？
感谢。

**Comment by redguardtoo** [comment, marked as answer]
```lisp
(with-eval-after-load 'pyim
  (setq pyim-pinyin-fuzzy-alist
        '(("en" "eng")
          ("in" "ing"))))
```

**Comment by fjlionj** [comment]
收到。感谢

    一个emacs初学者
：）

2022年6月10日 下午8:32，Chen Bin ***@***.******@***.***>> 写道：



(with-eval-after-load 'pyim
  (setq pyim-fuzzy-pinyin-alist
        '(("en" "eng")
          ("in" "ing"))))

**Comment by redguardtoo** [comment]
@fjlionj ,不好意思，好像pyim改了变量名（应该是`pyim-pinyin-fuzzy-alist`），我的设置代码太久没更新了，原答案已更新。
